visualisation/meshViewer.cpp

```diff
@@ -110,6 +110,45 @@ using namespace DGtal;
  */
 
 
+/**
+ * Custom Viewer3D to override KeyPressEvent method and handle new key display.
+ * It also desactivate the double Rendering mode for more efficiency.
+ **/
+template<typename Space = SpaceND<3>, typename KSpace = KhalimskySpaceND<3> >
+class CustomViewer3D: public Viewer3D<Space, KSpace>
```

Comment:
It maybe not necessayry to have a templated class : you can define CustomViewer3D as :
    class CustomViewer3D: public Viewer3D<>
et utiliser le Space et KSpace de Viewer3D (Viewer3D<>::Space et Viewer3D<>::KSpace) dans la redefinition des fonctions.